Sofa/framework/Geometry/src/sofa/geometry/Edge.h

```diff
@@ -121,6 +121,54 @@ struct Edge
     }
 
 
+    /**
+    * @brief	Test if a point is on Edge (n0, n1)
+    * @tparam   Node iterable container
+    * @tparam   T scalar
+    * @param	p0: position of the point to test
+    * @param	n0,n1: nodes of the edge
+    * @return	bool result if point is on Edge.
+    */
+    template<typename Node,
```

Comment:
Node sound a strange name for something iterable. I'm not sure this is the best name here;
I would have expected 
https://en.wikipedia.org/wiki/Vertex_(geometry)

Reply:
This is the name that has been chosen for the Geometry package at SOFA NG era.
I think it comes from Caribou structures.